core/includes/database/query.inc

```diff
@@ -184,37 +188,37 @@ interface QueryAlterableInterface {
   /**
    * Determines if a given query has a given tag.
    *
-   * @param $tag
+   * @param string $tag
    *   The tag to check.
    *
-   * @return
+   * @return bool
    *   TRUE if this query has been marked with this tag, FALSE otherwise.
    */
   public function hasTag($tag);
 
   /**
    * Determines if a given query has all specified tags.
    *
-   * @param $tags
+   * @param string ...$tags
    *   A variable number of arguments, one for each tag to check.
    *
-   * @return
+   * @return bool
    *   TRUE if this query has been marked with all specified tags, FALSE
    *   otherwise.
    */
-  public function hasAllTags();
+  public function hasAllTags(...$tags);
```

Comment:
Wow, this syntax for variadic functions has already been added in PHP 5.6. Phew...

Just adding this comment here, so the next reviewer doesn't also have to search all PHP changelogs for that variable argument token. :wink: 
https://wiki.php.net/rfc/variadics

Reply:
Thanks @indigoxela! Yeah I had to look up if we could use that syntax too. Looks like it's okay PHP 5.2+. Thanks for checking!